Backend/ManagerLayer/UserManagement/UserManagementManager.cs

```diff
@@ -34,9 +34,9 @@ public User CreateUser(string email, Guid SSOID)
             {
                 var useremail = new System.Net.Mail.MailAddress(email);
             }
-            catch (Exception)
+            catch (FormatException e)

```

Comment:
Catching the specific exception that MailAddress throws and then wrapping it inside the Email exception for specificity in the error handlers.